buildspec.yml

```diff
@@ -18,4 +18,13 @@ phases:
       - docker build -t $IMAGE_URI .
   post_build:
     commands:
-      - docker push $IMAGE_URI
\ No newline at end of file
+      - curl -fsSL https://raw.githubusercontent.com/usdot-jpo-codehub/aws-codebuild-extras/master/install >> extras.sh
+      - . ./extras.sh 
+      - echo $CODEBUILD_GIT_BRANCH
+      - |
+        if [ "$CODEBUILD_GIT_BRANCH" = "master" ] || [ "$CODEBUILD_GIT_BRANCH" = "development" ]; then
+          docker push "$IMAGE_URI:$IMAGE_TAG";
+          docker tag "$IMAGE_URI:$IMAGE_TAG" "$IMAGE_URI:latest"
+          docker push "$IMAGE_URI:latest";
+          aws ecs update-service --cluster $ECS_CLUSTER --service $ECS_SERVICE --force-new-deployment;
```

Comment:
Lets remove this line `aws ecs update-service --cluster $ECS_CLUSTER --service $ECS_SERVICE --force-new-deployment;` since we are using scheduled services now. The scheduled service will always pull the `latest` tag when it runs, so there's no redeployment necessary.

Reply:
done